x-pack/legacy/plugins/transform/public/app/sections/transform_management/components/transform_list/action_stop.tsx

```diff
@@ -4,45 +4,45 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React, { FC } from 'react';
+import React, { FC, useContext } from 'react';
 import { i18n } from '@kbn/i18n';
 import { EuiButtonEmpty, EuiToolTip } from '@elastic/eui';
 
-import { DataFrameTransformListRow, DATA_FRAME_TRANSFORM_STATE } from '../../../../common';
+import { TransformListRow, TRANSFORM_STATE } from '../../../../common';
 import {
-  checkPermission,
-  createPermissionFailureMessage,
-} from '../../../../../privilege/check_privilege';
+  createCapabilityFailureMessage,
+  AuthorizationContext,
+} from '../../../../lib/authorization';
 import { stopTransforms } from '../../services/transform_service';
 
 interface StopActionProps {
-  items: DataFrameTransformListRow[];
+  items: TransformListRow[];
   forceDisable?: boolean;
 }
 
 export const StopAction: FC<StopActionProps> = ({ items, forceDisable }) => {
   const isBulkAction = items.length > 1;
-  const canStartStopDataFrame: boolean = checkPermission('canStartStopDataFrame');
-  const buttonStopText = i18n.translate('xpack.ml.dataframe.transformList.stopActionName', {
+  const { canStartStopTransform } = useContext(AuthorizationContext).capabilities;
+  const buttonStopText = i18n.translate('xpack.transform.transformList.stopActionName', {
     defaultMessage: 'Stop',
   });
 
   // Disable stop action if one of the transforms is stopped already
   const stoppedTransform = items.some(
-    (i: DataFrameTransformListRow) => i.stats.state === DATA_FRAME_TRANSFORM_STATE.STOPPED
+    (i: TransformListRow) => i.stats.state === TRANSFORM_STATE.STOPPED
   );
 
   let stoppedTransformMessage;
   if (isBulkAction === true) {
     stoppedTransformMessage = i18n.translate(
-      'xpack.ml.dataframe.transformList.stoppedTransformBulkToolTip',
+      'xpack.transform.transformList.stoppedTransformBulkToolTip',
       {
-        defaultMessage: 'One or more selected data frame transforms is already stopped.',
+        defaultMessage: 'One or more selected transforms is already stopped.',
```

Comment:
As before, switch `is` to `are` and consider removing `selected`

Reply:
Fixed in ecc2c59.